.github/workflows/release-please.yaml

```diff
@@ -20,3 +21,36 @@ jobs:
           default-branch: ${{ steps.extract_branch.outputs.branch }}
           changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false},{"type":"refactor","section":"Miscellaneous","hidden":false},{"type":"test","section":"Miscellaneous","hidden":false},{"type":"doc","section":"Documentation","hidden":false}]'
           bump-minor-pre-major: true
+      - name: Extract release version
+        shell: bash
+        run: |
+          tag_name=$(echo "${{ steps.release.outputs.tag_name }}")
+          version=$(echo "${tag_name#v}")
+          echo "version=${version}" >> $GITHUB_OUTPUT
+        id: extract_version
+      - name: Trigger dispatch event - Enterprise
+        uses: peter-evans/repository-dispatch@v3
+        # release please run 2 times, first for creating the PR and second for creating the release
+        if: ${{ steps.release.outputs.releases_created == 'true' }}
+        with:
+          token: "${{ secrets.PAT }}"
+          repository: rudderlabs/rudderstack-operator
+          event-type: release-server-hosted
```

Comment:
event type should be release-rudder-server rt? as per [here](https://github.com/rudderlabs/rudderstack-operator/blob/a12bb0666a7a039fb3e2fcce8fd8e49185092598/.github/workflows/release-server.yaml#L4)
same for multitenant below

Reply:
yes, updated now